packages/syft/src/syft/core/common/serde/deserialize.py

```diff
@@ -9,6 +10,7 @@ def _deserialize(
     blob: Any,
     from_proto: bool = True,
     from_bytes: bool = False,
+    class_type: Type = type(None),
```

Comment:
I can't seem to find where this is being used. What is it for?

Reply:
so in order to store the state of an instantiated policy inside the database without too much effort, I am inserting the serializable decorator in the code string to use that. However, whenever I try to deserialize on code executed with `exec`, it seems that the new type cannot be found in the type bank. I didn't invest too much time ensuring that the type bank is correctly updated in this niche situation, as I already know what type I need. I use this optional argument to directly pass the type for user submitted policies at https://github.com/OpenMined/PySyft/blob/01ee892a0693180a0fa97de2ed649c8af7f06091/packages/syft/src/syft/core/node/new/policy.py#L202